app/main.py

```diff
@@ -1,3 +1,64 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        return (
+            Distance(self.km + other.km)
+            if isinstance(other, Distance)
+            else Distance(self.km + other)
+        )
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        self.km += other.km if isinstance(other, Distance) else other
+        return self
+
+    def __mul__(self, other: Distance | int | float) -> Distance:
```

Comment:
In mul and truediv Distance is imposible
Remove Distance only for this method

Reply:
done